app/controllers/application_controller.rb

```diff
@@ -1,2 +1,26 @@
 class ApplicationController < ActionController::API
+
+  SECRET_KEY = Rails.application.credentials.secret_key
+
+  def encode_token(payload,exp:Time.now+24.hours)
```

Comment:
follow proper indentation everywhere in the code. 
keep a blank space after comma and colon